### ISSUE TITLE
How to find out what path a program is trying to read? (`Deno requests read access to <exec_path>`)

### ISSUE THREAD

**Issue by josephrocca** [denoland/deno#19864]
I'm trying to use `npm:rollbar` and I'm running with `--allow-read=.` (amongst other permissions) but it's trying to read a file outside the current directory, and so it's giving me this:

```
┌ ⚠️  Deno requests read access to <exec_path>.
├ Requested by `Deno.execPath()` API.
├ Run again with --allow-read to bypass this prompt.
└ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all read permissions)
```

I'd like to know what file it's trying to read. That might help me report this to the Rollbar devs in case this read isn't really necessary, or they're doing something in a weird way. And ideally in the meantime I could allow access to just that specific path, rather than passing `--allow-read`, which is dangerous.

Using latest Deno version on Ubuntu 22.04.

**Comment by sigmaSd** [comment]
It's trying to read deno location 
you can use console.log(Deno.execPath()) to see the location of deno, and then you can pass that to the cli flags

**Comment by josephrocca** [comment]
@0f-0b Thank you!! I'll close this since it's not a problem with Deno. I've made a rollbar issue for this here: https://github.com/rollbar/rollbar.js/issues/1113

**Comment by sigmaSd** [comment]
This is not a rollbar.js problem, this is just how deno polyfills node process argv https://github.com/denoland/deno/blob/7e1218cd8f9856319e2c86e761e869ef48bf9c5f/ext/node/polyfills/process.ts#L715,  any npm package that needs argv will require the same permission

**Comment by josephrocca** [comment]
@sigmaSd So I guess the best approach (to limit permissions appropriately) here is something like this?
```
--allow-read=$(which deno)
```

**Comment by sigmaSd** [comment]
Yes, I have opened an issue before for an idea to improve this https://github.com/denoland/deno/issues/16766
